specification/healthbot/resource-manager/Microsoft.HealthBot/stable/2020-10-20/healthbot.json

```diff
@@ -0,0 +1,763 @@
+          "type": "string"
+        }
+      }
+    },
+    "Resource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "x-ms-azure-resource": true,
+      "properties": {
+        "id": {
+          "description": "Fully qualified resource Id for the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "name": {
+          "description": "The name of the resource",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of the resource.",
+          "type": "string",
+          "readOnly": true
+        },
+        "systemData": {
+          "$ref": "#/definitions/SystemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource"
+        }
+      }
+    },
+    "TrackedResource": {
+      "description": "The resource model definition for a ARM tracked top level resource",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "x-ms-mutability": [
+            "read",
+            "create",
+            "update"
+          ],
+          "description": "Resource tags."
+        },
+        "location": {
+          "type": "string",
+          "x-ms-mutability": [
+            "read",
+            "create"
+          ],
+          "description": "The geo-location where the resource lives"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "HealthBot": {
+      "type": "object",
+      "description": "HealthBot resource definition",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/HealthBotProperties",
+          "description": "The set of properties specific to healthcare bot resource."
+        }
+      }
+    },
+    "HealthBotProperties": {
+      "title": "HealthBotProperties",
+      "description": "The properties provided for bot creation. The Health Bot Service is a cloud platform that empowers developers in Healthcare organizations to build and deploy their compliant, AI-powered virtual health assistants and health bots, that help them improve processes and reduce costs.",
+      "properties": {
+        "planId": {
+          "type": "string",
+          "description": "The identifier of the plan (tier) the bot has.",
+          "enum": [
+            "free",
+            "standard"
+          ]
+        },
+        "provisioningState": {
+          "type": "string",
+          "description": "Gets the provisioning state of the Healthcare bot resource.",
+          "readOnly": true
+        }
+      }
+    },
+    "HealthBotUpdateProperties": {
```

Comment:
both "tags" and "sku" must also be patchable

Reply:
Fixed.